### ISSUE TITLE
[bug] Touching the ball with wheels doesn't reset touch timer

### ISSUE THREAD

**Issue by crimsoncoyote** [fabianvf/SpeedDrillPlugin#4]
Touching with wheels, and even getting a flip reset doesn't seem to reset the timer.

**Comment by fabianvf** [comment]
yeah, touching the ball with the bottom of the car doesn't register as a touch, haven't yet found a detector for it
